package.json

```diff
@@ -44,9 +45,14 @@
   "homepage": "https://github.com/4ver/node-auto-launch",
   "dependencies": {
     "applescript": "^1.0.0",
+    "gulp-rimraf": "^1.0.0",
     "mkdirp": "^0.5.1",
+    "natives": "^1.1.6",
```

Comment:
Is this needed?

Reply:
Doesn't seem so. Fixed by https://github.com/Oxalin/node-auto-launch/commit/e4e637dc1e94173cc04995a17e9a54664d1e9d4a